src/components/Inputs/EmailInput/EmailInput.js

```diff
@@ -27,6 +27,7 @@ export const EmailInput = (props) => {
         optional={optional}
         type="email"
         data-tid={restProps['data-tid']}
+        disabled={disabled}
```

Comment:
This was just missing so we couldn't pass through disabling of element down to the TextInput email delegates too.